rhizome/host/lib/vm_setup.rb

```diff
@@ -473,6 +474,17 @@ def generate_swap_config(swap_size_bytes)
   end
 
   def write_user_data(unix_user, public_keys, swap_size_bytes)
+    params = JSON.parse(File.read(vp.prep_json))
+    runcmd_nft_add = if params["boot_image"].include?("almalinux")
+<<NFT_SETUP
+  - [dnf, install, '-y', nftables]
```

Comment:
I feel like `bootcmd` runs in very early stage, and runs before `runcmd`. How `nft` commands work in `bootcmd`?

https://www.linode.com/docs/guides/run-shell-commands-with-cloud-init/#run-commands-with-bootcmd-directive

Reply:
IF nftables already installed, they run properly in bootcmd.